### ISSUE TITLE
R console memory

### ISSUE THREAD

**Issue by QuantumInformation** [holgerbrandl/r4intellij#131]
If I run a command then press up or down, it would be nice to be able to cycle previously written commands:

**Comment by holgerbrandl** [comment]
Yep. I was hoping to get this for free by using `AbstractConsoleRunnerWithHistory` but somehow it's not yet allowing to navigate the REPL history. Thanks for reporting it.
